### PR TITLE
add video conan in practice

### PR DESCRIPTION
**Pull request by solvingj**

### CODE REVIEW

File: videos.rst [hunk 1]
```diff
@@ -14,6 +14,14 @@
 Videos and links
 =================
 
+- Conan in Practice 2021: Live Demonstrations at C++ Meetup San Diego
+
+  .. raw:: html
+
+      <iframe width="560" height="315" src="" frameborder="0" allow="autoplay; encrypted-media; allowfullscreen>
```
Comment (lasote) [inline videos.rst:21]: Is it a valid link or...?

**Review by memsharded — CHANGES_REQUESTED** [review]
Seems missing link, please check @solvingj

**Comment by solvingj** [comment]
I see, will fix ASAP

**Comment by solvingj** [comment]
Please review @memsharded @lasote

**Comment by uilianries** [comment]
Jenkins is angry because Bintray is still listed in Docs.

**Review by uilianries — APPROVED** [review]
Screenshot_2021-05-04 Videos and links — conan 1 36 0 documentation

**Review by memsharded — APPROVED** [review]

**Comment by uilianries** [comment]
Let's rebuild, as #2094 has been merged

**Comment by uilianries** [comment]
Need to rebase

**Comment by solvingj** [comment]
Isn't there normally a github button for that? I dont see it here.

**Comment by uilianries** [comment]
Nope, only local git commands, not even github CLI offers rebase feature.

**Comment by solvingj** [comment]
rebased onto current master, lets see how it goes.

**Comment by uilianries** [comment]
I would say to target this PR into master branch.

**Comment by solvingj** [comment]
Sorry, please let this be the last thing lol

**Comment by solvingj** [comment]
@czoido is there anything we can do so our CI never fails with this again?

```
(line   50) broken    https://github.com/conan-io/examples/tree/master/features - 429 Client Error: too many requests for url: https://github.com/conan-io/examples/tree/master/features
(line    6) broken    https://github.com/plougher/squashfs-tools - 429 Client Error: too many requests for url: https://github.com/plougher/squashfs-tools
(line  112) broken    https://github.com/lasote/clion-conan-consumer - 429 Client Error: too many requests for url: https://github.com/lasote/clion-conan-consumer
(line  229) broken    https://github.com/lasote/clion-conan-package - 429 Client Error: too many requests for url: https://github.com/lasote/clion-conan-package
```

**Pull request merged**